tests/test_filterDiaSourceCatalog.py

```diff
@@ -258,7 +328,7 @@ def test_pixelScale_calculation(self):
         scale = filterDiaSourceCatalogTask._estimate_pixel_scale(self.diaSourceCat)
         # Should be almost but not actually equal
         self.assertNotEqual(self.config.estimatedPixelScale, scale)
-        self.assertAlmostEqual(self.config.estimatedPixelScale, scale)
+        self.assertAlmostEqual(self.config.estimatedPixelScale, scale, places=6)
```

Comment:
Why add this commit? The tests pass without it.

Reply:
It wasn't passing at the default threshold after my changes, though I didn't retry after the most recent changes. I'll run another test!

Reply:
Still failing without that change. :(
`FAILED tests/test_filterDiaSourceCatalog.py::TestFilterDiaSourceCatalogTask::test_pixelScale_calculation - AssertionError: 0.2 != np.float64(0.19999993377350445) within 7 places (np.float64(6.622649556531357e-08) difference)`